priv/build/index.html

```diff
@@ -54,9 +54,10 @@
      }
 
      .legend-label {
-       min-width: 400px !important;
-       padding: 10px;
-       margin-right: 10px;
+       font-size: 10px;
+       width: 100px !important;
```

Comment:
Do we have to apply `!important` here? There is no other way (e.g. specificity override)?